requirements.txt

```
@@ -0,0 +1,3 @@
+dpcpp_cpp_rt==2021.1
+daal==2021.1
+numpy==1.20

```

Users can use numpy less:
```suggestion
numpy>=1.15
```

```